coq/coq-syntax.el

```diff
@@ -246,6 +246,7 @@ so for the following reasons:
      ("move after" "mov" "move # after #" t "move")
      ("multimatch with" "mm" "multimatch # with\n| # => #\nend")
      ("multi_match! with" "multi_m" "multi_match! # with\n| # => #\nend")
+     ("native_compute" "nac" "native_compute." t "native_compute")
```

Comment:
Thanks @txyyss !

Can you please remove the trailing dot here?
```suggestion
     ("native_compute" "nac" "native_compute" t "native_compute")
```
so that it is similar to other declarations.

Reply:
Sure. It is done now. Thank you.